scripts/fetch-data

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+set -e
+
+if [[ -n "${IOW_BOUNDARY_TOOL_DEBUG}" ]]; then
+    set -x
+fi
+
+MUNICIPAL_BOUNDARIES_DATA_LOCATION="https://linc.osbm.nc.gov/explore/dataset/municipalities-2020/download/?format=geojson&lang=en"
+MUNICIPAL_BOUNDARIES_FILENAME=muni.json
```

Comment:
Consider naming this file `muni.geojson` to better communicate the kind of file it is (unless this leads to WebPack import issues?)

Reply:
It does cause WebPack import issues. What about `muni.geo.json`?

Reply:
Yes that sounds good!